Makefile

```diff
@@ -1,10 +1,65 @@
-test:
+# COLORS
+RED=\033[0;31m
+GREEN=\033[0;32m
+ORANGE=\033[0;33m
+BLUE=\033[0;34m
+PURPLE=\033[0;35m
+CYAN=\033[0;36m
+LIGHTGRAY=\033[0;37m
+DARKGRAY=\033[1;30m
+YELLOW=\033[0;33m
+NC=\033[0m # No Color
+
+chip?=F4
+BSPConfig?=off
+
+# Used to select the MCU (default is STM32F413)
+ifeq ($(chip), F4)
+    MCU_TARGET = STM32F413
+else ifeq ($(chip), L4)
+    MCU_TARGET = STM32L431
+else
```

Comment:
Could you echo the selected target after this if block? Would be nice to know what I'm building if I chose the default option